store/poi.go

```diff
@@ -14,11 +15,10 @@ type POIResourceRating struct {
 }
 
 func (m *mongoAccountStore) SetPOIRating(ctx context.Context, poiID string, ratings map[string]float64) error {
-
 	_, err := m.Resource("poi_ratings").UpdateOne(ctx,
 		bson.M{"id": poiID},
 		bson.M{
-			"$set":         bson.M{"ratings": ratings},
+			"$set":         bson.M{"ratings": ratings, "timestamp": time.Now()},
```

Comment:
```suggestion
			"$set":         bson.M{"ratings": ratings, "timestamp": time.Now().UTC()},
```

Reply:
I checked that the timestamp will always convert into UTC time. I am sure if there is any magic underneath.

Reply:
OK, the driver does the work. https://docs.mongodb.com/manual/tutorial/model-time-data/

Reply:
I turns the type of timestamp from `time.Time` to int64 in milliseconds. This prevents some time conversion problems between server communication and the zero problem of golang `time.Time`.